src/promptflow/promptflow/executor/_line_execution_process_pool.py

```diff
@@ -60,84 +61,21 @@ def persist_flow_run(self, run_info: FlowRunInfo):
-        self.output_queue = output_queue
-
-        # Put a start message and wait the subprocess be ready.
-        # Test if the subprocess can receive the message.
-        input_queue.put("start")
-
-        current_log_context = LogContext.get_current()
-        process = self.context.Process(
-            target=_process_wrapper,
-            args=(
-                self._executor_creation_func,
-                input_queue,
-                output_queue,
-                current_log_context.get_initializer() if current_log_context else None,
-                OperationContext.get_instance().get_context_dict(),
-            ),
-            # Set the process as a daemon process to automatically terminated and release system resources
-            # when the main process exits.
-            daemon=True,
+def format_current_process(process_name, pid, line_number: int, is_completed=False, is_failed=False):
+    if is_completed:
+        bulk_logger.info(f"Process name: {process_name}, Process id: {pid}, Line number: {line_number} completed.")
+    elif is_failed:
+        bulk_logger.info(f"Process name: {process_name}, Process id: {pid}, Line number: {line_number} failed.")
+    else:
+        bulk_logger.info(
+            f"Process name: {process_name}, Process id: {pid}, Line number: {line_number} start execution."
         )
 
-        self.process = process
-        process.start()
-
-        try:
-            # Wait for subprocess send a ready message.
-            output_queue.get(timeout=30)
-            self.is_ready = True
-        except queue.Empty:
-            self.end()
-            # If there are no more tasks, the process is not re-created
-            if not task_queue.empty():
-                self.start_new(task_queue)
-
-    def end(self):
-        # When process failed to start and the task_queue is empty.
-        # The process will no longer re-created, and the process is None.
-        if self.process is None:
-            return
-        if self.process.is_alive():
-            self.process.kill()
-
-    def put(self, args):
-        self.input_queue.put(args)
-
-    def get(self):
-        return self.output_queue.get(timeout=1)
-
-    def format_current_process(self, line_number: int, is_completed=False):
-        process_name = self.process.name if self.process else None
-        process_pid = self.process.pid if self.process else None
-        if is_completed:
-            bulk_logger.info(
-                f"Process name: {process_name}, Process id: {process_pid}, Line number: {line_number} completed."
-            )
-        else:
-            bulk_logger.info(
-                f"Process name: {process_name}, Process id: {process_pid}, Line number: {line_number} start execution."
-            )
-
-        return f"Process name({process_name})-Process id({process_pid})-Line number({line_number})"
+    return f"Process name({process_name})-Process id({pid})-Line number({line_number})"
```

Comment:
May not be a good practice to "print something in a string format function".

Reply:
Add log_process_status function to print.'
The function 'format_current_process' focus on format current process.